### PR TITLE
Verify a Credential with SSI Service

### PR DESCRIPTION
**Pull request by bobbilee19**

### CODE REVIEW

**Comment by bobbilee19** [comment]
@angiejones

1. Should I now add the following to the VC creation guide...after Getting Credentails section..
```
### Verifying Credentials
To build upon this tutorial, copy your newly created VC's `credentialJwt` field to [verify your credential](verify-credentials) like so:

```bash
curl -X PUT localhost:8080/v1/credentials/verification -d '{
    "credentialJwt": "eyJhbGciOiJFZERTQSIsImtpZCI6ImRpZDprZXk6..."
}'
```
```
2. Presentation Exchange section should I add this sentence after the first paragraph?
`The SSI Service supports Presentation Exchange. Its usage will be covered in another guide coming soon.`

File: site/docs/ssi/verify-credentials.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+sidebar_position: 4
+title: Verify Credentials
+hide_title: true
+---
+
+# Verify a Verifiable Credential (VC)
```
Comment (angiejones) [inline site/docs/ssi/verify-credentials.md:7]: such a mouthful haha. can we change to Verify a Credential?

File: site/docs/ssi/verify-credentials.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+There are numerous factors to determine a credential's validity. Verifiers will ultimatily define their own unique criteria for determining validity from a range of possibilities:
+
+- **Signature Verification** - Signed by the issuer? Has the signature been tampered with?
```
Comment (angiejones) [inline site/docs/ssi/verify-credentials.md:11]: Can we put the description in sentence form? Example:

- **Signature Verification** - Has it been signed by the issuer? If so, has the signature been tampered with?
- **Credential Status** - Is it still active, and not revoked or suspended?
- **Credential Validity** - Has it expired?
- **Credential Issuer** - Is the issuer trusted for the claims they're attesting to?
- **Credential Evidence** - Is there evidence supporting the claims being made?
- **Credential Schema** - Does it conform to a schema defined in the `credentialSchema` property?

File: site/docs/ssi/verify-credentials.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+- **Credential Schema** - Does it conform to a schema defined in the `credentialSchema` property?
+- And more...
+
+:::note
+Determining the validity of a VC can be a complex topic. The data model itself [has guidance on validity checks](https://www.w3.org/TR/vc-data-model/#validity-checks), and a [separate section on validation](https://www.w3.org/TR/vc-data-model/#validation), which provide even more information for implementers to consider.
+:::
+
+<Divider type="slash" />
+
+## **Verify the VC**
```
Comment (angiejones) [inline site/docs/ssi/verify-credentials.md:25]: nit: when you use ## it makes an H2, so you don't also need to use ** which makes it bold

File: site/docs/ssi/verify-credentials.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+}
+```
+
+The request above performs the following verification process:
+
+- Make sure the credential is complaint with the VC Data Model
+- Make sure the credential is not expired
+- Make sure the signature of the credential is valid (currently supports both JWT and some Linked Data credentials)
+- If the credential has a schema, makes sure its data complies with the schema (note: the schema must be hosted within the service)
+- In the future this endpoint can (and should!) be expanded to support status checks and external schema resolution, among other optional checks.
```
Comment (angiejones) [inline site/docs/ssi/verify-credentials.md:57]: make the last bullet an info section instead:

```
:::info
In the future, this endpoint will likely be expanded to support status checks and external schema resolution, as well as other optional checks.
:::
```

File: site/docs/ssi/verify-credentials.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+## Other Types of Verification
+
+### Verifiable Presentations
+The example we've gone through above verifies a credential from an issuer. But what about verifying the presentation of a credential, or set of credentials, from a holder to a verifier? To do this, a holder must construct what's called a [Verifiable Presentation](https://www.w3.org/TR/vc-data-model/#presentations-0), an object which is also defined by the VC Data Model, which allows a holder of a verifiable credential to create an authenticated wrapper around a set of credentials it wishes to present to a verifier. Learn more in our [guide on presentations](https://github.com/TBD54566975/ssi-service/blob/main/doc/howto/presentation.md).
```
Comment (angiejones) [inline site/docs/ssi/verify-credentials.md:62]: change wording in the final sentence:

"Learn more in the guide on Verifying a Presentation"

File: site/docs/ssi/verify-credentials.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+### Presentation Exchange
+What about applying more complex logic to the verification process? Like checking if a credential was issued from a known set of issuers? Or requesting two of one type of credential and three of another? Or checking that certain credential fields are present and have expected values? With [Presentation Exchange](https://identity.foundation/presentation-exchange/), a specification created in the Decentralized Identity Foundation this arbitarily-complex style of verification is made possible.
```
Comment (angiejones) [inline site/docs/ssi/verify-credentials.md:65]: Add comma after "Foundation"

**Review by angiejones — CHANGES_REQUESTED** [review]

**Comment by angiejones** [comment]
> @angiejones
> 
> 1. Should I now add the following to the VC creation guide...after Getting Credentails section...
> 
> ```
> ### Verifying Credentials
> To build upon this tutorial, copy your newly created VC's `credentialJwt` field to [verify your credential](verify-credentials) like so:
> 
> ```bash
> curl -X PUT localhost:8080/v1/credentials/verification -d '{
>     "credentialJwt": "eyJhbGciOiJFZERTQSIsImtpZCI6ImRpZDprZXk6..."
> }'
> ```
> 
> 2. Presentation Exchange section should I add this sentence after the first paragraph?
> 
> `The SSI Service supports Presentation Exchange. Its usage will be covered in another guide coming soon.`

1. nah, we should make a new guide for that. I'll add a task to do so. Should also make a separate guide for Getting Credentials
2. left a note here: https://github.com/TBD54566975/developer.tbd.website/pull/644#discussion_r1289093647

**Review by angiejones — APPROVED** [review]

**Pull request merged**